lib/src/excel.dart

```diff
@@ -19,6 +19,13 @@ class ExcelDart {
   static const _horizontalAlignment = 'Center';
   static const _borderStyle = 'Continuous';
   static const _currencyFormat = '\$#,##0.00';
+  static const _badFont = '#9C0006';
+  static const _badFill = '#FFC7CE';
+
+  static const _spentColumn = '\$N2';
```

Comment:
Add a comment to say that this has to match with the table header & body declaration.